packages/shared/lib/core/profile/interfaces/persisted-profile.interface.ts

```diff
@@ -20,6 +20,6 @@ export interface IPersistedProfile {
     lastUsedAccountIndex?: number
     clientOptions: IClientOptions
     forceAssetRefresh: boolean
-    strongholdVersion: StrongholdVersion
+    strongholdVersion?: StrongholdVersion
```

Comment:
if you make this optional, ideally you should also update `packages/shared/lib/core/profile/actions/active-profile/saveActiveProfile.ts`